### ISSUE TITLE
only use it on 2D size

### ISSUE THREAD

**Issue by squarewhiteflag** [chao1224/GraphMVP#21]
I want to use the model GraphMVP using a 2D dataset to pretrain(classfication task ) , how can I do that?In the repo I can use your code such as GraphLoG to run my baseline

**Comment by chao1224** [comment]
Hi @squarewhiteflag , do you mean only 2D topology is available?

**Comment by squarewhiteflag** [comment]
Yes.My understanding is that graphMVP uses 3D topology for pre-training and then finetune downstream tasks without 3D topology. The dataset used is 3D. But I only use 2D datasets for pre-training, so I modify the loss of the code to make the code run, and fine-tune it in downstream task for testing.It seems reasonable for the data I got from downstream task. Is this the right thing to do? Sorry for the late reply

**Comment by chao1224** [comment]
Hi @squarewhiteflag,

For GraphMVP, it supports:
- pretraining on 2D & 3D
- downstream on 2D or 3D
    - We didn't try 3D downstream because of the insufficient 3D pretraining dataset. Now we have added them in [Geom3D](https://github.com/chao1224/Geom3D)


If you only have a 2D dataset for pretraining, the objective functions proposed in GraphMVP (i.e., EBM-NCE and VRR) still hold, and only the application setting changes.

**Comment by squarewhiteflag** [comment]
I got it.Thank you very much
